core/amber/src/main/scala/edu/uci/ics/texera/web/model/websocket/request/TexeraWebSocketRequest.scala

```diff
@@ -10,7 +10,6 @@ import edu.uci.ics.texera.web.model.websocket.request.python.{
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
 @JsonSubTypes(
   Array(
-    new Type(value = classOf[RegisterWorkflowIdRequest]),
```

Comment:
you also need to remove the corresponding frontend code that sends this request right??

Reply:
removed.